src/plugins/intel_cpu/src/node.h

```diff
@@ -584,6 +587,7 @@ class Node {
     std::vector <dnnl::memory::format_tag> inputMemoryFormatsFilter;
     std::vector <dnnl::memory::format_tag> outputMemoryFormatsFilter;
     bool enforceBF16evenForGraphTail = false;
+    bool accuracyAware = false;
```

Comment:
can we call it `keepOriginalPrecision` or smt like that? For me it sounds more meaningful.

Reply:
Applied.